packages/dds/tree/src/test/shared-tree/fuzz/targetedFuzz.spec.ts

```diff
@@ -20,6 +21,7 @@ export async function performFuzzActionsAbort(
 	generator: AsyncGenerator<Operation, FuzzTestState>,
 	seed: number,
 	saveInfo?: SaveInfo,
+	savedInfo?: SavedInfo
```

Comment:
Once you make the change I suggested (to have the file data come from the generator), this function won't need this special parameter.
Same for the other fuzz test file.